zanata-war/src/main/java/org/zanata/webtrans/server/HibernateIntegrator.java

```diff
@@ -37,6 +38,8 @@ public void integrate(Configuration configuration,
                     updateListener);
             eventListenerRegistry.appendListeners(EventType.POST_INSERT,
                     updateListener);
+            eventListenerRegistry.appendListeners(EventType.POST_COMMIT_UPDATE,
+                    ServiceLocator.instance().getInstance(SlugEntityUpdatedListener.class));
```

Comment:
Do you mean inside the POJO listener, use ServiceLocator to get access to seam beans?